source/features/sticky-sidebar.tsx

```diff
@@ -38,5 +39,6 @@ void features.add(__filebasename, {
 	init,
 	deinit: () => {
 		observer.disconnect();
+		selectObserver.abort();
 	}
```

Comment:
Both of these functions should be added to a deinit array instead

Reply:
I does not work 😢 

I get `Illegal invocation`